tests/test_wps_CI.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+from pkg_resources import resource_filename
+from tempfile import NamedTemporaryFile
+
+from wps_tools.testing import run_wps_process
+from chickadee.processes.wps_CI import CI
+
+
+@pytest.mark.online
```

Comment:
What online resource are we using? Both of the files you target are local.

Reply:
Wow, I think it became my habit to mark wps tests as online. I will fix that.